src/doc.test.ts

```diff
@@ -0,0 +1,102 @@
+import { isValid } from "@govtechsg/oa-verify";
+import { verify } from "./issuers-verifier";
+import {
+    openAttestationVerifiers,
+    verificationBuilder,
+    VerificationFragmentType,
+    VerificationManagerOptions,
+    Verifier,
+    CodedError,
+  } from "@govtechsg/oa-verify";
+  
+const doc: any = {
```

Comment:
lazy to type ?

Reply:
Lol stop reviewing, this is not for review yet, I’m rushing out